src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Device.Gpio;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    private const string LibgpiodLibrary = "System.Device.Gpio.Native";
+
+    /// <summary>
+    ///     Create a new gpiochip iterator.
+    /// </summary>
+    /// <returns>Pointer to a new chip iterator object or NULL if an error occurred.</returns>
+   [DllImport(LibgpiodLibrary, SetLastError = true)]
+   internal static extern SafeChipIteratorHandle GetChipIterator();
+
+   /// <summary>
+   ///     Release all resources allocated for the gpiochip iterator and close the most recently opened gpiochip(if any).
+   /// </summary>
+   /// <param name="iter">The gpiochip iterator object</param>
+   [DllImport(LibgpiodLibrary)]
+   internal static extern void FreeChipIterator(IntPtr iter);
```

Comment:
Should this be a SafeChipIteratorHandle?

Reply:
SafeHandle uses IntPtr for closing resources